opendebates/views.py

```diff
@@ -171,6 +174,7 @@ def vote(request, id):
 
     if request.method == "GET":
         two_other_approved_ideas = list(Submission.objects.filter(
+            category__site_mode=request.site_mode,
```

Comment:
Is this necessary?  We already have `category=idea.category` in the filter so hopefully it's redundant.